contracts/disputes/DisputeManager.sol

```diff
@@ -611,14 +639,24 @@ contract DisputeManager is DisputeManagerV1Storage, GraphUpgradeable, IDisputeMa
      * @notice Ignore a dispute with ID `_disputeID`
      * @param _disputeID ID of the dispute to be disregarded
      */
-    function drawDispute(bytes32 _disputeID) external override onlyArbitrator {
-        Dispute memory dispute = _resolveDispute(_disputeID);
+    function drawDispute(bytes32 _disputeID)
+        public
+        override
+        onlyArbitrator
+        onlyPendingDispute(_disputeID)
+    {
+        Dispute storage dispute = disputes[_disputeID];
+
+        // store dispute status
+        dispute.status = IDisputeManager.DisputeStatus.Drawn;
 
         // Return deposit to the fisherman
         TokenUtils.pushTokens(graphToken(), dispute.fisherman, dispute.deposit);
 
         // Resolve the conflicting dispute if any
-        _resolveDisputeInConflict(dispute);
+        if (_isDisputeInConflict(dispute)) {
+            drawDispute(dispute.relatedDisputeID);
```

Comment:
This will end up pushing the tokens twice. I think you can leave this part as it was before, and simply change _resolveDisputeInConflict to mark the conflicting dispute as Drawn